apps/dashboard/src/components/workflow-editor/steps/email/maily.tsx

```diff
@@ -98,9 +98,15 @@ export const Maily = (props: MailyProps) => {
                       editor?.commands.deleteRange({ from, to });
                       editor?.commands.insertContent({
                         type: 'variable',
-                        attrs: { id: queryWithoutSuffix, label: null, fallback: null, showIfKey: null },
+                        attrs: {
+                          id: queryWithoutSuffix,
+                          label: null,
+                          fallback: null,
+                          showIfKey: null,
+                          required: false,
```

Comment:
question: What the required prop controls here?

Reply:
The ⚠️ icon in Maily jargon. For now, all of our variables should not be required.

Reply:
But does is solve the actual problem then? Because the issue was happening when maily autocomplete returned "empty results" instead of allowing to add a new variable from the dropdown

Reply:
A database migration solves the issue. I was torn between teaching the code to handle invalid payload vs just correcting the DB value and I did the latter as for code-first any user-defined payload schema should be accepted.